### PR TITLE
Implement player_info

### PR DESCRIPTION
**Pull request by kannadan**

### CODE REVIEW

File: server/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/server/impl/PlayerInfoRouter.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                            ctx.bodyAsJson
+                        } catch (e: DecodeException) {
+                            throw UserError("Invalid json ${e.message}")
+                        }
+                        body.fieldNames().forEach { key ->
+                            if (!playerInfo.containsKey(key)) {
+                                throw UserError("Unknown key in request: $key")
+                            }
+                        }
+                        // val newData: PlayerInfo = existingPlayerInfo.copy(message = body.message)
```
Comment (Moetto) [inline server/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/server/impl/PlayerInfoRouter.kt:69]: Remove commented out code

**Review by Moetto — CHANGES_REQUESTED** [review]
Very nice

File: server/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/server/impl/PlayerInfoRouter.kt (hunk 1 above)
Comment (kannadan) [reply server/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/server/impl/PlayerInfoRouter.kt]: Removed

**Comment by Moetto** [comment]
Run `./gradlew spotlessApply`

**Review by Moetto — APPROVED** [review]

**Pull request merged**